sdk/storage/storage-blob/test/containerclient.spec.ts

```diff
@@ -222,7 +222,7 @@ describe("ContainerClient", () => {
     }
   });
 
-  it("Verify AsyncIterator for listBlobsFlat", async () => {
+  it("Verify AsyncIterator(generator .next() syntax) for listBlobsFlat", async () => {
```

Comment:
would the special characters `(` and `)` affect later record and playback test generation?

Reply:
No, they won't. We handled those things gracefully 😊